pkgs/top-level/all-packages.nix

```diff
@@ -1735,15 +1735,43 @@ with pkgs;
 
   circleci-cli = callPackage ../development/tools/misc/circleci-cli { };
 
-  citrix_receiver = callPackage ../applications/networking/remote/citrix-receiver { };
-  citrix_receiver_13_10_0 = citrix_receiver.override { version = "13.10.0"; };
-  citrix_receiver_13_9_1  = citrix_receiver.override { version = "13.9.1";  };
-  citrix_receiver_13_9_0  = citrix_receiver.override { version = "13.9.0";  };
-  citrix_receiver_13_8_0  = citrix_receiver.override { version = "13.8.0";  };
-  citrix_receiver_13_7_0  = citrix_receiver.override { version = "13.7.0";  };
-  citrix_receiver_13_6_0  = citrix_receiver.override { version = "13.6.0";  };
-  citrix_receiver_13_5_0  = citrix_receiver.override { version = "13.5.0";  };
-  citrix_receiver_13_4_0  = citrix_receiver.override { version = "13.4.0";  };
+  citrix_receiver_unwrapped = callPackage ../applications/networking/remote/citrix-receiver { };
+  citrix_receiver_unwrapped_13_10_0 = citrix_receiver_unwrapped.override { version = "13.10.0"; };
+  citrix_receiver_unwrapped_13_9_1  = citrix_receiver_unwrapped.override { version = "13.9.1";  };
+  citrix_receiver_unwrapped_13_9_0  = citrix_receiver_unwrapped.override { version = "13.9.0";  };
+  citrix_receiver_unwrapped_13_8_0  = citrix_receiver_unwrapped.override { version = "13.8.0";  };
+  citrix_receiver_unwrapped_13_7_0  = citrix_receiver_unwrapped.override { version = "13.7.0";  };
+  citrix_receiver_unwrapped_13_6_0  = citrix_receiver_unwrapped.override { version = "13.6.0";  };
+  citrix_receiver_unwrapped_13_5_0  = citrix_receiver_unwrapped.override { version = "13.5.0";  };
+  citrix_receiver_unwrapped_13_4_0  = citrix_receiver_unwrapped.override { version = "13.4.0";  };
```

Comment:
For the next release we should drop versions older then 13.8, since there support will be dropped before 19.03: https://www.citrix.co.uk/support/product-lifecycle/milestones/receiver.html